wizards/PolicyAutomation/PolicyAutomationWizard.tsx

```diff
@@ -220,6 +226,17 @@ export function PolicyAutomationWizard(props: {
                             }
                         }}
                     />
+                    <WizNumberInput
+                        hidden={(item) => item.spec?.mode !== 'everyEvent'}
+                        path="spec.delayAfterRunSeconds"
+                        label="DelayAfterRunSeconds"
+                        labelHelp="DelayAfterRunSeconds is the minimum seconds before an automation can be restarted on the same cluster. 
+                        When a policy is violated, the automation runs one time in the delay period, after which it runs every time for a new violation. 
+                        If the policy is violated multiple times during the delay period and kept in the violated state, 
+                        the automation runs one time in the delay period and another time after it. 
+                        The default is 0 seconds and is only applicable for the everyEvent mode."
```

Comment:
@dockerymick could you please review this suggestion?
```suggestion
                        labelHelp="DelayAfterRunSeconds is the minimum seconds before an automation can be restarted on the same cluster. 
                        When a policy is violated, the automation runs one time before the delay period.
                        If the policy is violated multiple times during the delay period and kept in the violated state, 
                        the automation runs one time after the delay period. 
                        The default is 0 seconds and is only applicable for the everyEvent mode."
```

Reply:
This suggestion looks good to me! @mprahl